modules/frontend/config.go

```diff
@@ -69,6 +69,7 @@ func (cfg *Config) RegisterFlagsAndApplyDefaults(string, *flag.FlagSet) {
 			MaxDuration:           168 * time.Hour, // 1 week
 			ConcurrentRequests:    defaultConcurrentRequests,
 			TargetBytesPerRequest: defaultTargetBytesPerRequest,
+			IngesterShards:        1,
```

Comment:
I thought about making the shards per-tenant, but I think this doesn't keep the query fairness.  This is probably fine for now, but we can discuss.